src/Tickets/Emails/Email_Abstract.php

```diff
@@ -828,4 +828,101 @@ public function get_post_type_data(): array {
 
 		return $data;
 	}
+
+	/**
+	 * Get a successful order array for preview.
+	 * 
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_preview_order_successful(): array {
+		return [
+			'provider'   => 'Stripe',
+			'status'     => 'success',
+			'post_title' => 'Black Midi with Special Guests Chat Pile and Apprehend',
+			'total'      => '$100.00',
+			'purchaser'  => [
+				'name' => 'John Doe',
+				'email' => 'john@doe.com',
+			],
+			'created' => 'March 1, 2023',
+			'id'      => 123,
```

Comment:
The strings that are for presentation purposes need to be translatable (Dates, post title, purchaser).

Reply:
I wasn't quite sure that this would turn out to be the final format of the arrays, so I didn't go as far as to make things translatable. I'll do that, though.